### ISSUE TITLE
Remove features that should not be used by authors

### ISSUE THREAD

**Issue by chasenlehara** [CCALI/a2jauthor#232]
As a user of assistive technology (AT), I do not want the preview document to include at-hostile markup, so that I can more easily consume the preview page.

Currently, authors can set `accesskey` and `tabindex` on links, but these markup features should not be used. If possible, they should be removed from the authoring tool.

**Comment by chasenlehara** [comment]
It doesn’t look like the attributes mentioned in the description can be easily removed, so I think we can provide guidance to authors to not use these options.
